spec/ic.did

```diff
@@ -112,12 +112,10 @@ service ic : {
   http_request : (record {
     url : text;
     max_response_bytes: opt nat64;
-    method : variant { get; head; post };
+    http_method : variant { GET; HEAD; POST };
```

Comment:
We will change the enumerations to lowercase in the implementation.
However, we wonder whether using "method" or "http_method" is better, considering that it is part of a record that has "http" in its name.

Reply:
I have a slight preference for just calling it "method" since it's in the context of an `http_request` and the HTTP spec refers to it as "request method" (not "HTTP method"). But I don't think it matters too much, I find it clear either way.

Reply:
Agreed, than let's go with
```
method : variant { get; head; post };
```
as it is currently in the spec and change the implementation.